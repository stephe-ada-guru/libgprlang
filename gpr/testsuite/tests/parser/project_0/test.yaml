driver: parser
rule: simple_project
