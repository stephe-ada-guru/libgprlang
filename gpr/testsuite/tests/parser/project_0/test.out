SimpleProject[1:1-2:12]
|name:
|  Id[1:9-1:15]
|  |tok: Simple
|items: <null>
|end_id:
|  Id[2:5-2:11]
|  |tok: Simple
